acceptancetests/clean_resources.py

```diff
@@ -27,28 +27,62 @@ def get_regions(args, env):
 
 def clean(args):
     env = SimpleEnvironment.from_config(args.env)
-    selected_regions = get_regions(args, env)
+    # To Do: Read regions from public-clouds.yaml
+    selected_regions = ['us-east-1', 'us-east-2', 'us-west-1', 'us-west-2',
+                        'ca-central-1',
+                        'eu-west-1', 'eu-west-2', 'eu-central-1',
+                        'ap-south-1', 'ap-southeast-1', 'ap-southeast-2',
+                        'ap-northeast-1', 'ap-northeast-2',
+                        'sa-east-1']
+    logging.info('The target regions for cleaning job are \n{}'
+                 .format('\n'.join(selected_regions)))
     for region in selected_regions:
         with AWSAccount.from_boot_config(env, region=region) as substrate:
-            logging.info('Cleaning resources in {}.'.format(substrate.region))
-            all_groups = dict(substrate.iter_security_groups())
-            instance_groups = dict(substrate.iter_instance_security_groups())
-            non_instance_groups = dict((k, v) for k, v in all_groups.items()
-                                       if k not in instance_groups)
-            try:
-                unclean = substrate.delete_detached_interfaces(
-                    non_instance_groups.keys())
-                logging.info('Unable to delete {} groups'.format(len(unclean)))
-            except:
-                logging.info('Unable to delete non-instance groups {}'.format(non_instance_groups.keys()))
-            for group_id in unclean:
-                logging.debug('Cannot delete {}'.format(all_groups[group_id]))
-            for group_id in unclean:
-                non_instance_groups.pop(group_id, None)
-            try:
-                substrate.destroy_security_groups(non_instance_groups.values())
-            except:
-                logging.debug('Failed to delete groups {}'.format(non_instance_groups.values()))
+            if substrate is not None:
+                logging.info('Cleaning resources in {}.'
+                             .format(substrate.region))
+                all_groups = dict(substrate.iter_security_groups())
+                instance_groups = dict(substrate.iter_instance_security_groups())
+                logging.info('{} items found from instance groups on {} \n{}'
```

Comment:
preferred breaking up of lines is to have the newline just after an opening paren.
So this would be:

logging.info(
    '{} items ...'.format(
        str(len(..)),
        ...))